.github/workflows/codeql.yml

```
@@ -0,0 +1,46 @@
+name: "CodeQL OpenKAT"
+
+on:
+  push:
+    branches: [main, release-*, feature/*]
+  pull_request:
+    branches: ["main"]
```

This means for every PR the action will run two times, one time because of the push to the feature/ branch and one time because it is a PR. This is what is configured in every other action and works correctly:

```suggestion
  push:
    branches:
      - "main"
      - "release-*"
  pull_request:
```